### PR TITLE
Fix: Travis troubles

### PR DESCRIPTION
**Pull request by CozyKourai**
Fix several problem on travis 
All fixes in on PR to really check Travis status (and easier for everyone to rebase after merge)
See trello links below for more info on thoses problems

OLD PR : 
- https://github.com/cozy/cozy-drive/pull/1594
- https://github.com/cozy/cozy-drive/pull/1590
- https://github.com/cozy/cozy-drive/pull/1595

### CODE REVIEW

**Comment by CozyKourai** [comment]
[Fix : ClientFunction execution was interrupted by page unload.](https://trello.com/c/a42pzKzL/1879-fix-clientfunction-execution-was-interrupted-by-page-unload)

**Comment by CozyKourai** [comment]
[Fix : No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.](https://trello.com/c/SR7xRmRl/1895-fix-no-output-has-been-received-in-the-last-10m0s-this-potentially-indicates-a-stalled-build-or-something-wrong-with-the-build-i)

**Comment by CozyKourai** [comment]
[Fix: TypeError: Converting circular structure to JSON](https://trello.com/c/KHKs0q6i/1899-fix-typeerror-converting-circular-structure-to-json)

**Comment by Crash--** [comment]
Si les soucis sont justes sur la liste des fichiers et les icônes qui changent allons y pour le merge. 

Il me semble que je les avais tous validés avec les nouveau éléments

File: testcafe/tests/helpers/utils.js [hunk 1]
```diff
@@ -119,27 +120,22 @@ export async function setDownloadPath(downloadFolderPath) {
...
+          resolve(response ? response.statusCode : 0)
+        })
+      })
+    )
   }
-
-  await t.expect(requestsCount).eql(statuses.length)
+  let statuses = await Promise.all(requestPromises)
+  console.log(statuses)
   for (const status of statuses) await t.expect(status).eql(200)
```
Comment (Crash--) [inline testcafe/tests/helpers/utils.js]: Console log à retirer

File: testcafe/tests/helpers/utils.js [hunk 2]
```diff
@@ -119,27 +120,22 @@ export async function setDownloadPath(downloadFolderPath) {
...
 
   for (let i = 0; i < count; i++) {
     let url = await images.nth(i).getAttribute('src')
-    requestsCount++
-    statuses.push(await getRequestResult(url))
+    if (!url.startsWith('http')) url = location + url
+    requestPromises.push(
+      new Promise(resolve => {
+        return request(location + url, function(error, response) {
+          resolve(response ? response.statusCode : 0)
```
Comment (Crash--) [inline testcafe/tests/helpers/utils.js]: Utiliser error et reject aussi

**Pull request merged**